Bluepill-cli/BPInstanceTests/BPTestHelper.m

```diff
@@ -66,15 +65,21 @@ + (NSString *)samplePhotoPath {
 }
 
 + (NSString *)bpExecutablePath {
-    return [[[[NSBundle bundleForClass:[self class]] bundlePath] stringByDeletingLastPathComponent] stringByAppendingPathComponent:@"bp"];
+    return [[self debugMacOSPath] stringByAppendingPathComponent:@"bp"];
 }
 
 #pragma mark - Helpers
 
 + (NSString *)derivedDataPath {
-    NSString *currentPath = [[NSBundle bundleForClass:[self class]] bundlePath];
-    return [[[currentPath stringByDeletingLastPathComponent]
-            stringByDeletingLastPathComponent] stringByAppendingPathComponent:@"Debug-iphonesimulator"];
+    return [[[[[[NSBundle bundleForClass:[self class]] bundlePath] stringByDeletingLastPathComponent] stringByDeletingLastPathComponent] stringByDeletingLastPathComponent] stringByDeletingLastPathComponent];
```

Comment:
This is very hard to read. Please add comment.

Reply:
added